### DISCUSSION TITLE
Make bot never leave the vc

### DISCUSSION THREAD

**Discussion by Prasham27** [eritislami/evobot#412] [Q&A]
How do I make the bot never leave vc even after the song ends, I wanna keep the bot 24/7 in vc, is that possible @eritislami @mihawkman

**Comment by eritislami** [comment]
https://github.com/eritislami/evobot/pull/351#issuecomment-737062973

**Reply by Prasham27** [reply]
It aint working, I created a variable in Heroku named STAY_TIME and literally spammed 9999 but it didnt work, can u help @eritislami or can u change the code which doesnt make the bot leave vc or any add-ons?

**Comment by WindSekirun** [comment]
* Open 'include/play.js' in your favorite editor (If doesn't exist, i recommand vscode)
* Delete 12~16 lines of that file
  ```
   setTimeout(function () {
        if (queue.connection.dispatcher && message.guild.me.voice.channel) return;
        queue.channel.leave();
        queue.textChannel.send("Leaving voice channel...");
      }, STAY_TIME * 1000);
  ```
* npm run start

**Reply by WindSekirun** [reply]
@PrashLucifer 

When I get back home, I removed setTimeout and it seems to work on my local machine (also, lightsail too.)

What you attached as a [crash log](https://github.com/eritislami/evobot/pull/351#issuecomment-737148038) seems to show up as a PermissionError, so it doesn't seem to be related to this problem.

**Reply by Prasham27** [reply]
> @PrashLucifer
> 
> When I get back home, I removed setTimeout and it seems to work on my local machine (also, lightsail too.)
> 
> What you attached as a [crash log](https://github.com/eritislami/evobot/pull/351#issuecomment-737148038) seems to show up as a PermissionError, so it doesn't seem to be related to this problem.

Can we talk on Discord? And can you help me out coding because I've been working on it since many days!

**Reply by WindSekirun** [reply]
@PrashLucifer English is not the first language, so it seems that it is only possible in text. If is ok, my discord nickname is 'P****' (will be masked after 10 minutes)
